litgpt/model.py

```diff
@@ -213,15 +264,15 @@ def __init__(self, config: Config, block_idx: int) -> None:
             )
 
         self.norm_1 = config.norm_class(config.n_embd, eps=config.norm_eps)
-        self.attn = CausalSelfAttention(config, block_idx)
         self.post_attention_norm = (
             config.norm_class(config.n_embd, eps=config.norm_eps) if config.post_attention_norm else nn.Identity()
         )
         self.norm_2 = None if config.shared_attention_norm else config.norm_class(config.n_embd, eps=config.norm_eps)
-        self.mlp = config.mlp_class(config)
         self.post_mlp_norm = (
             config.norm_class(config.n_embd, eps=config.norm_eps) if config.post_mlp_norm else nn.Identity()
         )
+        self.attn = CausalSelfAttention(config, block_idx)
+        self.mlp = config.mlp_class(config)
```

Comment:
Could you move them back where they were before?

Reply:
OK